routes/index.js

```diff
@@ -1,9 +1,9 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
```

Comment:
```suggestion
const express = require("express");
const router = express.Router();
```
`var`를 `const`로 변경해주시면 좋을 것 같습니다!

Reply:
적용 완료하였습니다